Application/EdFi.Ods.Profiles.Sample/EdFi.Ods.Profiles.Sample.nuspec

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
```

Comment:
This nuspec can probably remain since it won't cause any issues.

Reply:
I kept this file . Thanks